src/main/java/pl/touk/sputnik/configuration/GeneralOption.java

```diff
@@ -22,6 +25,7 @@
 
     SCORE_STRATEGY("score.strategy", "Score strategy: <NoScore|ScoreAlwaysPass|ScorePassIfEmpty|ScorePassIfNoErrors>", "ScoreAlwaysPass"),
     SCORE_PASSING_KEY("score.passingKey", "Score passing key", "Code-Review"),
+    SCORE_PASSING_COMMENT("score.passingComment", "Comment when no errors are found", "Perfect!"),
```

Comment:
It's not well documented (of course). Scoring is for gerrit comments on a patchset. score package fits better as options, while this option would fit better to messages package, like messages.perfectComment.